src/System.IO.Compression.Brotli/System/IO/Compression/Brotli/BrotliStream.cs

```diff
@@ -307,19 +308,19 @@ public override void Write(byte[] buffer, int offset, int count)
                 Marshal.Copy(buffer, currentOffset, BufferIn, copyLen);
                 bytesRemain -= copyLen;
                 currentOffset += copyLen;
-                AvailIn = (IntPtr)copyLen;
+                AvailIn = (nuint)copyLen;
                 NextIn = BufferIn;
                 while ((int)AvailIn > 0)
                 {
                     if (!BrotliNative.BrotliEncoderCompressStream(_encoder.State, BrotliNative.BrotliEncoderOperation.Process, ref AvailIn, ref NextIn, ref AvailOut,
                         ref NextOut, out totalOut)) throw new System.IO.IOException("Unable compress stream"); 
```

Comment:
Is it mean to create file with string constants and use it instead of "Magic strings"? May be it'll be better to create some new specialized exceptions?